CONTRIBUTING.md

```diff
@@ -1,155 +0,0 @@
-# Contributing to Chef
```

Comment:
Yeah, no big deal, I wouldn't block too long on figuring it out. I'm happy to defer to @tas50, but the Github feature is nice.

https://github.com/blog/1184-contributing-guidelines